src/commands/base-command.ts

```diff
@@ -665,6 +666,7 @@ export default class BaseCommand extends Command {
       globalConfig,
       // state of current site dir
       state,
+      frameworksAPIPaths: getFrameworksAPIPaths(buildDir, this.workspacePackage),
```

Comment:
Moved this to the base command, so that you can easily access this from anywhere without having to duplicate the call to `getFrameworksAPIPaths`.